_includes/podstatus.html

```diff
@@ -115,15 +56,13 @@ <h2 class="leaderboard">LeaderBoard</h2>
     var team_scores = { Team_1: 0, Team_2: 0, Team_3: 0, Team_4: 0 , Team_5: 0, Team_6: 0, Team_7: 0};
 
     async function getApi(url) {
-  // Storing response
-  const response = await fetch(url);
-
-  // Storing data in form of JSON
-  var data = await response.json();
-  renderData(data);
+      // getting the date from api as response
```

Comment:
Why is podstatus file changed in this PR? Please make sure the PR only focus on the custom scrollbar. If you need to fix some other things, you can create a separate PR